### PR TITLE
Adding my project Hackernews-Scraping

### PR DESCRIPTION
**Pull request by pushp1997**
I think it's good project for beginners in OSS contribution and web scraping. Which makes it a good fit for up for grabs.

Let me know if you have any feedback related to the template.

### CODE REVIEW

File: _data/projects/thehackernews-scraping.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Hackernews Scraping
+desc: Web Scraper written in Python, that scrapes thehackernews.com upto specified number of pages.
```
Comment (shiftkey) [inline _data/projects/thehackernews-scraping.yml]: Typo:

```suggestion
desc: Web scraper written in Python, that scrapes thehackernews.com up to specified number of pages.
```

**Review by shiftkey — COMMENTED** [review]
Thanks for the contribution @pushp1997!

Just one suggestion around the description itself - everything else looks 💎 !

**Comment by pushp1997** [comment]
> Thanks for the contribution @pushp1997!
> 
> Just one suggestion around the description itself - everything else looks 💎 !

Thanks a ton @shiftkey  😄

**Review by shiftkey — APPROVED** [review]

**Comment by pushp1997** [comment]
Thank you for accepting the pull request! @shiftkey

**Pull request merged**